parse_gmn_rootfiles.C

```diff
@@ -4,6 +4,9 @@
 #include <string> 
 #include <cstring>
 #include <dirent.h>
+#include "TStopwatch.h"
+using namespace std::chrono;
```

Comment:
Hi John, instead of declaring the whole script to be within the namespace "std::chrono", could you explicitly do it like: `std::chrono::<the function you want to call>`. I think that is better to avoid namespace collisions.